hackathon_site/dashboard/frontend/src/testing/mockData.js

```diff
@@ -105,3 +105,17 @@ export const teamCode = "PAS3NLQ3";
 // Navbar
 export const cartQuantity = 0;
 export const userEmail = "graham@email.com";
+
+// Inventory
+export const categories = [
```

Comment:
Might want to give this a more detailed name like `inventoryCategories`, just in case we have more than one kind of categories in this file eventually